src/core/core.ticks.js

```diff
@@ -49,7 +49,9 @@ module.exports = {
 				var maxTick = Math.max(Math.abs(ticks[0]), Math.abs(ticks[ticks.length - 1]));
 				if (maxTick < 1e-4) { // all ticks are small numbers; use scientific notation
 					var logTick = helpers.log10(Math.abs(tickValue));
-					tickString = tickValue.toExponential(Math.floor(logTick) - Math.floor(logDelta));
+					var numExponential = Math.floor(logTick) - Math.floor(logDelta);
+					numExponential = Math.max(Math.min(numExponential, 20), 0);
```

Comment:
Why 0-20? I see in the [error message](https://github.com/CARTAvis/carta-frontend/issues/433) that you got that valid values are 0-100, so I'm wondering if it might make sense to use that range

Reply:
Firstly, because it's consistent with the code below, and secondly because MDN mentions that range in its [description](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toExponential)